etc/containerpilot.json

```diff
@@ -1,5 +1,5 @@
 {
-  "consul": "{{ .CONSUL }}:8500",
+  "consul": "localhost:8500",
```

Comment:
We will want to check the env variable CONSUL_AGENT to determine where to connect to..

```
"consul": "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}{{ end }}:8500",
```